modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/impl/IngestServiceImpl.java

```diff
@@ -1481,13 +1495,30 @@ public void discardMediaPackage(MediaPackage mp) throws IOException {
     logger.info("Sucessful discarded mediapackage {}", mp);
   }
 
+  /**
+   * Creates a StandAloneTrustedHttpClientImpl
+   *
+   * @param user
+   * @param password
+   * @return
+   */
+  protected TrustedHttpClient createStandaloneHttpClient(String user, String password) {
+    return new StandAloneTrustedHttpClientImpl(this.downloadUser, this.downloadPassword, none(), none(), none());
```

Comment:
If the `user` parameter is null here there will be an IllegalArgumentException thrown.  I'm not sure what would happen with a null password, but I'd guess nothing good :)